tests/jballerina-unit-test/src/test/resources/test-src/expressions/lambda/iterable/basic-iterable-with-variable-mutability.bal

```diff
@@ -64,81 +64,75 @@ function testBasicArray2(string[] values) returns string {
-//                                  if (v == "a" || v == "e" && count != 0) {
-//                                      count = count + 10;
-//                                      return true;
-//                                  }
-//                                  return false;
-//                            });
-//    return [count, values];
-//}
-//
-//function testBasicMap2() returns [int, string, string[]] {
-//    int count = 0;
-//    string str = "start";
-//    map<string> m = {a:"A", b:"B", c:"C", d:"D", e:"E"};
-//    string[] values = m.entries().'map(function ([string, string] tuple) returns [string, string] {
-//                                var [key, value] = tuple;
-//                                str = str + "-"  + key + " : " + value;
-//                                count = count + 10;
-//                                return [key, value];
-//                            })
-//                      .filter(function ([string, string] v) returns boolean {
-//                                  var [k, t] = v;
-//                                  if (k == "a" || k == "e") {
-//                                      count = count + 10;
-//                                      return true;
-//                                  }
-//                                  return false;
-//                              })
-//                      .'map(function ([string, string] v) returns string {
-//                               var [v1, v2] = v;
-//                               count = count + 10;
-//                               return v1 + v2;
-//                           });
-//    str = str + "-end";
-//    count = count + 1;
-//    return [count, str, values];
-//}
-
-// XML WITH NESTED ITERABLE OPERATIONS
-// Uncomment after implementing xml.map langlib function.
-//
-//function xmlTest() returns [string, map<any>] {
-//    string str = "start";
-//    xml xdata = xml `<p:person xmlns:p="foo" xmlns:q="bar">
-//        <p:name>bob</p:name>
-//        <p:address>
-//            <p:city>NY</p:city>
-//            <q:country>US</q:country>
-//        </p:address>
-//        <q:ID>1131313</q:ID>
-//    </p:person>`;
-//
-//    int index = 0;
-//    map<xml> m = xdata.*.elements()[1].*.elements()
-//                 .'map(function (xml|string x) returns [string, xml] {
-//                            index += 1;
-//                            if x is xml {
-//                                str = str + "-" + string.convert(index);
-//                                return [string.convert(index), x];
-//                            }
-//                            return ["", xml ` `];
-//                      });
-//    str = str + "-end";
-//    return [str, m];
-//}
+function testBasicMap1() returns [int, string[]] {
+    map<string> m = {a:"A", b:"B", c:"C", d:"D", e:"E"};
+    int count = 0;
+    string[] values = m.keys().'map(function (string key) returns string {
+                                count = count + 10;
```

Comment:
Indentation seems to be incorrect here. The indentation should start from where the `function` starts.
Eg - 
```
string[] values = m.keys().'map(function (string key) returns string {
                                    count = count + 10;
```

@NipunaRanasinghe WDYT? Please check whether this formatting change can be supported via the IDEA(and LS) in the future if seems good.

Reply:
@Shan1024 Yeah +1. It would be better if the function body can be indented w.r.t. where the function starts, as Shan has suggested. :) 

@NipunaMarcus WDYT? Should we define these sub-rules in [Ballerina Style Guide](https://ballerina.io/learn/style-guide/definitions/#function-definition)?

Reply:
@NipunaRanasinghe we don't have this specific rule ... we have a general rule how we should handle line breaks ... when a line break happens we indent the broken line with four spaces related to the parent of it.